pkgs/servers/http/router/default.nix

```diff
@@ -10,16 +10,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "router";
-  version = "1.30.1";
+  version = "1.45.1";
```

Comment:
Is it possible to bump to 1.49.1?

https://github.com/apollographql/router/releases/tag/v1.49.1

Reply:
Some tests were failing with 1.49.1, so I wanted to fix the vulnerability asap.

Reply:
I made a quick a test with 1.49.1 with your current changes, it seems to build correctly?

Reply:
Maybe the problem was only on my side. I'll try the latest and see if CI is fine with it.